src/github.com/getlantern/pac/pac2go.sh

```diff
@@ -18,13 +18,12 @@ BINPATH=../pac-cmd/binaries
 
 mv $BINPATH/windows/pac.exe $BINPATH/windows/pac
 osslsigncode sign -pkcs12 "$BNS_CERT" -pass "$BNS_CERT_PASS" -in $BINPATH/windows/pac -out $BINPATH/windows/pac || die "Could not sign windows"
-go-bindata -nomemcopy -nocompress -pkg pac -prefix $BINPATH/windows -o pac_bytes_windows.go $BINPATH/windows    
+go-bindata -nomemcopy -nocompress -pkg pac -prefix $BINPATH/windows -o pac_bytes_windows.go $BINPATH/windows
 mv $BINPATH/windows/pac $BINPATH/windows/pac.exe
 
 go-bindata -nomemcopy -nocompress -pkg pac -prefix $BINPATH/linux_386 -o pac_bytes_linux_386.go $BINPATH/linux_386
 go-bindata -nomemcopy -nocompress -pkg pac -prefix $BINPATH/linux_amd64 -o pac_bytes_linux_amd64.go $BINPATH/linux_amd64
-go-bindata -nomemcopy -nocompress -pkg pac -prefix $BINPATH/linux_arm -o pac_bytes_linux_arm.go $BINPATH/linux_arm
+#go-bindata -nomemcopy -nocompress -pkg pac -prefix $BINPATH/linux_arm -o pac_bytes_linux_arm.go $BINPATH/linux_arm
```

Comment:
@uaalto If you can find the binary, you can add it and uncomment this line of the script.  As it stands, we still have the pac_bytes_linux_arm.go from whenever that got generated, so people can still run Lantern on arm.  Until then, I have to leave this commented-out since the script won't run successfully otherwise.